account/identity.go

```diff
@@ -161,9 +161,9 @@ func (m *GormIdentityRepository) Create(ctx context.Context, model *Identity) er
 		}, "unable to create the identity")
 		return errs.WithStack(err)
 	}
-	log.Info(ctx, map[string]interface{}{
+	log.Debug(ctx, map[string]interface{}{
```

Comment:
I suggest to just reduce the test output by setting `F8_LOG_LEVEL=error` instead of changing the level code-wise. Most repo create functions use `log.Info`. Also run tests with `go test` instead of `go test -v`

Reply:
I know I can do that (`F8_LOG_LEVEL=error` and `go test` without the `verbose` flag), but do we really need those logs statements at the `INFO` level ?

Reply:
Probably not. Did you cover all "created X" log messages? Then it is fine to change to Debug if you ask me. I mostly feared mixing concerns of this PR.